airflow/providers/amazon/aws/operators/eks.py

```diff
@@ -224,6 +229,7 @@ def __init__(
         wait_for_completion: bool = False,
         aws_conn_id: str = DEFAULT_CONN_ID,
         region: str | None = None,
+        deferrable: bool = False,
```

Comment:
After https://github.com/apache/airflow/pull/31712 is merged, we might want to use that feature and change the default value to the following.

```suggestion
        deferrable: bool = conf.getboolean("operators", "default_deferrable", fallback=False),,
```